File: backend/orion/api/interactive/case_manager/case_config.py
from orion.services.mongo_manager.shared_model.db_case_model import CaseStatus


CASE_STATUS_FLOW = [
    CaseStatus.NEW,
    CaseStatus.INTAKE_REVIEW,
    CaseStatus.UNDER_INVESTIGATION,
    CaseStatus.EVIDENCE_COLLECTION,
    CaseStatus.VERIFICATION,
    CaseStatus.REGULATORY_ACTION,
    CaseStatus.LEGAL_REVIEW,
    CaseStatus.RESOLVED,
    CaseStatus.CLOSED,
]

File: client/src/app/pages/user-management/sidebar-user-case-management/model/case.config.ts
import { CaseStatus } from './case.model';

export const CASE_STATUS_WORKFLOW: { value: CaseStatus; label: string }[] = [
  { value: 'new', label: 'New' },
  { value: 'intake_review', label: 'Intake Review' },
  { value: 'under_investigation', label: 'Under Investigation' },
  { value: 'evidence_collection', label: 'Evidence Collection' },
  { value: 'verification', label: 'Verification' },
  { value: 'regulatory_action', label: 'Regulatory Action' },
  { value: 'legal_review', label: 'Legal Review' },
  { value: 'resolved', label: 'Resolved' },
  { value: 'closed', label: 'Closed' }
];